chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/RoomCustomDataSpek.kt

```diff
@@ -0,0 +1,177 @@
+package com.pusher.chatkit
+
+import com.google.common.truth.Truth.assertThat
+import com.pusher.chatkit.Rooms.GENERAL
+import com.pusher.chatkit.Rooms.SAMPLE_CUSTOM_DATA
+import com.pusher.chatkit.Users.ALICE
+import com.pusher.chatkit.Users.PUSHERINO
+import com.pusher.chatkit.Users.SUPER_USER
+import com.pusher.chatkit.test.InstanceActions.createDefaultRole
+import com.pusher.chatkit.test.InstanceActions.newRoom
+import com.pusher.chatkit.test.InstanceActions.newUsers
+import com.pusher.chatkit.test.InstanceSupervisor.setUpInstanceWith
+import com.pusher.chatkit.test.InstanceSupervisor.tearDownInstance
+import com.pusher.chatkit.util.FutureValue
+import junit.framework.Assert.assertNotNull
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.it
+
+object RoomCustomDataSpek : Spek({
+    beforeEachTest(::tearDownInstance)
```

Comment:
It's more clear to have tearDown actually after test. It also leaves the instance clean at the end. Having it after `closeChatManagers` would be good.

Reply:
We already have `afterEachTest(::closeChatManagers)` which is L22 - is that what you meant? I kept whatever we had existing in the `RoomSpek` tests.

Reply:
That's what I mean:
```kotlin
    afterEachTest(::closeChatManagers)
    afterEachTest(::tearDownInstance)
```

Reply:
Ah, doing both things after each test :) That makes sense. Have refactored!